jupyterhub/values.schema.yaml

```diff
@@ -247,6 +247,22 @@ properties:
                   - jovyan2
           ```
 
+          __Example__
+
+          If your kubernetes pods take longer than usual to spawn, the users notebooks
+          may fail to spawn with an error like `<user>'s server failed to start in 30000 
+          seconds, giving up`. In such scenarios, you may need to adjust 
+          [KubeSpawner defaults](https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html#kubespawner.KubeSpawner.k8s_api_request_timeout)
+          which can also be done using `hub.config`. For example, 
```

Comment:
Yeah usually it definitely takes even less than 3 seconds. However, I've noticed that since April, EKS has updated some configuration regarding its ports, which sometimes makes the requests take much longer than even 30 seconds (Hence me needing to look into this option in the first place). I am still not sure on what the change is, but noticed that an EKS cluster updated after March had the issue, while the ones without the update were working fine... So not sure on the exact issue there. 
Regardless, yes, 3600 is definitely VERy far fetched 😅

Reply:
As far as your update goes, it looks all good 👍.

Reply:
Thank you for the followp and help improving the docs!!! Let's go for a merge!

> However, I've noticed that since April, EKS has updated some configuration regarding its ports, which sometimes makes the requests take much longer than even 30 seconds (Hence me needing to look into this option in the first place)

Hmmm hmmm I suspect something is up with the EKS cluster's DNS or similar. I've worked with lots of k8s clusters and I think its a big issue if a simple request to create a Pod resource or similar (which returns before its started etc) should be taking less than 1s.